docs/readthedocs/source/doc/Orca/QuickStart/orca-pytorch-quickstart.md

```diff
@@ -116,34 +116,32 @@ First, Create an Estimator
 from bigdl.orca.learn.pytorch import Estimator 
 from bigdl.orca.learn.metrics import Accuracy
 
-est = Estimator.from_torch(model=model, optimizer=adam, loss=criterion, metrics=[Accuracy()], backend="bigdl")
+est = Estimator.from_torch(model=model_creator, optimizer=optim_creator, loss=nn.NLLLoss(), metrics=[Accuracy()], use_tqdm=True)
 ```
 
 Next, fit and evaluate using the Estimator
 
 ```python
-from bigdl.orca.learn.trigger import EveryEpoch 
+est.fit(data=train_loader_creator, epochs=1, batch_size=batch_size)
+```
 
-est.fit(data=train_loader, epochs=10, validation_data=test_loader,
-        checkpoint_trigger=EveryEpoch())
+Finally, evaluate using the Estimator.
```

Comment:
remove this? as evaluate is already mentioned in `Next, fit and evaluate using the Estimator`